utils/urlUtils.js

```diff
@@ -0,0 +1,18 @@
+function getUrlDetails(url) {
+    let urlDetails = {
+        project : "",
+        type : "",
+        id : ""
+    }
+
+    url = url.split('/')
+    position = url.indexOf('github.com')
+
+    urlDetails.project = url[position+2]
+    urlDetails.type = url[position+3]
+    urlDetails.id = url[position+4]
+
+    return urlDetails;
+}
```

Comment:
if github.com is in `upperCase` then this would fail. (Although you can't really enter `https://GITHUB.com/coding-blocks/boss/issues/29923232`. this throws `duplicate issue error` (a bug maybe?). But still it's better to make sure it doesn't break in the future) 

Also `project`, `type` should be converted to `lowerCase`.

`https://github.com/coding-blocks/BOSS/issues/29923232`
` https://github.com/coding-blocks/boss/issues/29923232` This fails when these two are the issueUrl. `Op.like` is case-sensitive I guess

Reply:
@prabalsingh24 'GITHUB.com' and 'github.com' should be fixed at add claim level. thanks for pointing the project, and type. yes, they should be converted to lower case.

Reply:
@prabalsingh24 @hereisnaman I think we just need to convert both issue URL and pull URL to lowercase before adding. Everything will work fine, no need to do any change anywhere else. just convert url to lowercase on adding.

Reply:
Yeah that would work but if there is already a pull url with uppercase, then it might fail in that case. I know chances of that happening is pretty low. But better to be on the safer side and fix it from the both side?

Reply:
we will add convert code to add claims and also write a migration to convert existing ones.